src/RazorToolingTestApp.LibraryPackage/project.json

```diff
@@ -1,7 +1,7 @@
 {
   "version": "1.0.0",
   "dependencies": {
-    "Microsoft.AspNetCore.Razor.Runtime": "1.1.0-*",
+    "Microsoft.AspNetCore.Razor.Runtime": "1.0.0",
```

Comment:
Ya we should. That can go in separately though. @ajaybhargavb could you file an issue to create cross-versioned tests?